dataikuapi/dss/ml.py

```diff
@@ -1752,6 +1752,84 @@ def get_diagnostics(self):
         diagnostics = self.details.get("trainDiagnostics", {})
         return [DSSMLDiagnostic(d) for d in diagnostics.get("diagnostics", [])]
 
+    def export_documentation(self, folder_id=None, path=None):
+        """
+        Start the export of the model documentation from a template docx file in a managed folder,
+        or from the default template if no folder id and path are specified.
+
+        :param folder_id: (optional) the id of the managed folder
+        :param path: (optional) the path to the file from the root of the folder
+        :return: A :class:`~dataikuapi.dss.future.DSSFuture` representing the model document generation process
+        """
+        if bool(folder_id) != bool(path):
+            raise ValueError("Both folder id and path arguments are required to use a template from folder. Use without argument to export using the default template")
+
+        export_mode_url = "default-template" if folder_id is None and path is None else "template-in-folder"
+
+        if self.mltask is not None:
+            f = self.mltask.client._perform_json(
+                "POST", "/projects/%s/models/lab/%s/%s/models/%s/export-documentation-from-%s" %
+                        (self.mltask.project_key, self.mltask.analysis_id, self.mltask.mltask_id, self.mltask_model_id, export_mode_url),
+                params={"folderId": folder_id, "path": path})
+            return DSSFuture(self.mltask.client, f["jobId"])
+        else:
+            f = self.saved_model.client._perform_json(
+                "POST", "/projects/%s/savedmodels/%s/versions/%s/export-documentation-from-%s" %
+                        (self.saved_model.project_key, self.saved_model.sm_id, self.saved_model_version, export_mode_url),
+                params={"folderId": folder_id, "path": path})
+            return DSSFuture(self.saved_model.client, job_id=f["jobId"])
+
+    def export_documentation_from_custom_template(self, fp):
+        """
+        Start the export of the model documentation from a docx template (as a file object).
+
+        :param object fp: A file-like object pointing to a template docx file
+        :return: A :class:`~dataikuapi.dss.future.DSSFuture` representing the model document generation process
+        """
+        files = {'file': fp}
+        if self.mltask is not None:
+            f = self.mltask.client._perform_json(
+                "POST", "/projects/%s/models/lab/%s/%s/models/%s/model-document/export-documentation-from-custom-template" %
```

Comment:
/model-document/export-documentation-from-custom-template seems not a valid URL

Reply:
Ah indeed 👍 changed the url but missed the extra `model-document/`